### PR TITLE
Refactoring of server deployment configs

### PR DESCRIPTION
**Pull request by TanVD**
1.Deployment scenario with all tomcat webapps on one tomcat created.
2.Ports and paths of all services now assigned in parent pom.xml

### CODE REVIEW

File: auth-service/pom.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```
Comment (jzuken) [inline auth-service/pom.xml:22]: why filtering here?

File: pom.xml [hunk 2]
```diff
@@ -43,6 +28,53 @@
...
+                <port.dashboard>8082</port.dashboard>
+                <path.editor>/editor</path.editor>
+                <path.auth>/auth</path.auth>
+                <path.dashboard>/dashboard</path.dashboard>
+                <path.editor.service>/editorRest</path.editor.service>
+                <path.dashboard.service>/dashboardRest</path.dashboard.service>
+            </properties>
+        </profile>
+        <profile>
+            <id>OneTomcatForAll</id>
```
Comment (jzuken) [inline pom.xml]: let's add 

```
<activation>
      <activeByDefault>true</activeByDefault>
</activation>
```

here to make it default profile

File: pom.xml [hunk 3]
```diff
@@ -43,6 +28,53 @@
         <port.db.user>9092</port.db.user>
     </properties>
 
+    <!-- You can use default ports and paths stating right profile, or add your own -->
+    <!-- Profiles state ports and paths only for tomcat based services for now -->
+    <profiles>
+        <profile>
+            <id>Travis</id>
```
Comment (jzuken) [inline pom.xml]: is there a reason profile names start with a capital letter?

Comment (TanVD) [reply pom.xml]: No reason. Will change it to lower-case style.

**Pull request merged**